js/foundation.offcanvas.js

```diff
@@ -351,7 +351,7 @@ class OffCanvas extends Plugin {
     }
     elem.allowUp = elem.scrollTop > 0;
     elem.allowDown = elem.scrollTop < (elem.scrollHeight - elem.clientHeight);
-    elem.lastY = event.originalEvent.pageY;
+    elem.lastY = event.touches[0].pageY;
```

Comment:
`event.originalEvent` is a valid property of the jQuery event object. See https://api.jquery.com/category/events/event-object/